clients/web/src/templates/layout/layoutHeader.pug

```diff
@@ -1,4 +1,4 @@
-.g-header-wrapper
+.g-header-wrapper(style='background-color:' + bannerColor)
```

Comment:
Since you're setting `background-color` here, please remove it from [the Stylus file](https://github.com/girder/girder/blob/master/clients/web/src/stylesheets/layout/header.styl#L4).